core-java-modules/core-java-locale/README.md

```diff
@@ -0,0 +1,6 @@
+ Core Java Cookbooks and Examples
```

Comment:
Please change the title ' Core Java Cookbooks and Examples' to ' Core Java Locale'

Reply:
done